lib/auth/autoupdate/autoupdatev1/service.go

```diff
@@ -292,6 +293,11 @@ func (s *Service) CreateAutoUpdateVersion(ctx context.Context, req *autoupdate.C
 		return nil, trace.Wrap(err)
 	}
 
+	if modules.GetModules().Features().Cloud && !isAdmin(authCtx) {
+		return nil, trace.AccessDenied("only role %q is allowed to modifying %q in cloud",
```

Comment:
This error is not very actionable for users. I think we should have something like:
> This Teleport instance is running on Teleport Cloud. The "autoupdate_version" resource is managed by the Teleport Cloud team. You can use the "autoupdate_config" resource to opt-in, opt-out or configure update schedules.